src/steps/curves-club.js

```diff
@@ -0,0 +1,38 @@
+import { Given, When, Then } from 'cucumber';
+
+Given('I visit the Curves club lingerie website', function() { 
+    browser.url('https://www.curvesclublingerie.com/');
+    // browser.pause('5000');
```

Comment:
It seems like you left some comments in your steps. Do you want to remove those?

Reply:
comments removed.